### ISSUE TITLE
Add trait: GDPR compliance

### ISSUE THREAD

**Issue by fabianekc** [decentralized-identity/did-traits#14]
In some use cases it is relevant to ensure GDPR compliance for the identifiers used. Therefore, I propose to discuss if this trait makes sense and might be added.

**Comment by jceb** [comment]
@fabianekc are there specific features that make a DID method GDPR compliant? The right to be forgotten comes to mind.

**Comment by jceb** [comment]
Example definition:

```json
{
    "gdpr-compliant": {
      "type": "boolean",
      "title": "GDPR-compliant",
      "description": "A DID method's ability to fully adhere to the European Union's General Data Protection Regulation (GDPR) requirements, ensuring comprehensive protection of personal data and individual privacy rights."
    }
}
```

**Comment by ottomorac** [comment]
It seems "did:tdw" is now called "did:webvh" per a post from Stephen Curran on the #wg-id slack channel in DIF.

**Comment by jceb** [comment]
The trait has been added. It's up to the method author to provide detailed information on the compliance.

**Comment by jceb** [comment]
In today's call, we concluded that this trait is too difficult to express in a simple boolean value and not technical enough. Therefore, the trait will be removed.

**Comment by ottomorac** [comment]
Yes agreed. As we discussed compliance with data protection can even be altered depending in how the did method is implemented in specific instance.
